### DISCUSSION TITLE
What does Triton actually do on Windows? Didn't notice any difference

### DISCUSSION THREAD

**Discussion by stepahin** [bmaltais/kohya_ss#2176] [Q&A]
Hi! As I understand it Triton has only recently become available on Windows and just a couple of weeks ago as an option in setup.dat. I installed it but didn't notice any difference. I've heard several times in Windows vs Linux discussions something like "Linux is faster to train because Triton is available there, it's not on Windows". So what does Triton actually do? How can I see its advantages and at what settings? Could find literally no info. Thanks!

**Comment by brianiup** [comment]
I tried installing it and after I did that now nothing will train, just exceptions at training start...
